test-runner/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,37 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Set everything to be logged to the file target/unit-tests.log
+test.appender=file
+log4j.rootCategory=INFO, ${test.appender}
+log4j.appender.file=org.apache.log4j.FileAppender
+log4j.appender.file.append=true
+log4j.appender.file.file=target/unit-tests.log
```

Comment:
A by-product of having the properties file in the submitted jar is getting logs when spark runs on slave to a file:
/tmp/mesos/slaves/41575072-681d-4eaa-a651-ba14fc4eebf9-S0/frameworks/41575072-681d-4eaa-a651-ba14fc4eebf9-0006/executors/driver-20151211110434-0001/runs/9c908929-961e-4bbb-9eaa-cd379891a4a4/spark-1.5.1-bin-hadoop2.6/target/unit-tests.log
This might be useful in debugging.